lib/JsonpMainTemplatePlugin.js

```diff
@@ -36,7 +36,7 @@ JsonpMainTemplatePlugin.prototype.apply = function(mainTemplate) {
 			"var script = document.createElement('script');",
 			"script.type = 'text/javascript';",
 			"script.charset = 'utf-8';",
-			"script.async = true;",
+			"script.async = false;",
```

Comment:
by default async flag is true. So no.
hmm, why it is work in v1.0
need additional investigation i think